medical_emr_data/__openerp__.py

```diff
@@ -36,20 +37,10 @@
 
     'name': 'Medical EMR: Module Data',
     'version': '1.0',
-    'author': "OeMEdical Team,Odoo Community Association (OCA)",
+    'author': "OeMEdical Team, LasLabs, Odoo Community Association (OCA)",
```

Comment:
To keep consistency with master medical module, I propose: Odoo Medical Team, ...